UIKitDZ/UIKitDZ/AddBirthdayViewController.swift

```diff
@@ -0,0 +1,278 @@
+         textField.placeholder = "Введите имя"
+
+         return textField
+     }()
+
+     private var dataLabel: UILabel = {
+         var label = UILabel()
+         label.text = "Дата"
+         label.textColor = #colorLiteral(red: 0.4902775884, green: 0.6610349417, blue: 0.9130681157, alpha: 1)
+         label.font = UIFont(name: "Arial Bold", size: 15)
+         label.frame = CGRect(x: 20, y: 280, width: 100, height: 15)
+         return label
+     }()
+
+     private lazy var dataTextField: UITextField = {
+         var textField = UITextField()
+         textField.frame = CGRect(x: 20, y: 300, width: 320, height: 30)
+         textField.placeholder = "Введите дату"
+
+         textField.inputView = dataPicker
+         return textField
+     }()
+
+     private var ageLabel: UILabel = {
+         var label = UILabel()
+         label.text = "Возраст"
+         label.textColor = #colorLiteral(red: 0.4902775884, green: 0.6610349417, blue: 0.9130681157, alpha: 1)
+         label.font = UIFont(name: "Arial Bold", size: 15)
+         label.frame = CGRect(x: 20, y: 360, width: 100, height: 15)
+         return label
+     }()
+
+     private lazy var ageTextField: UITextField = {
+         var textField = UITextField()
+         textField.frame = CGRect(x: 20, y: 380, width: 320, height: 30)
+         textField.placeholder = "Добавить"
+
+         textField.inputView = agePicker
+         return textField
+     }()
+
+     private var sexLabel: UILabel = {
+         var label = UILabel()
+         label.text = "Пол"
+         label.textColor = #colorLiteral(red: 0.4902775884, green: 0.6610349417, blue: 0.9130681157, alpha: 1)
+         label.font = UIFont(name: "Arial Bold", size: 15)
+         label.frame = CGRect(x: 20, y: 440, width: 100, height: 15)
+         return label
+     }()
+
+     private lazy var sexTextField: UITextField = {
+         var textField = UITextField()
+         textField.frame = CGRect(x: 20, y: 460, width: 320, height: 30)
+         textField.placeholder = "Добавить"
+
+         textField.inputView = sexPicker
+         return textField
+     }()
+
+     private var instagramLabel: UILabel = {
+         var label = UILabel()
+         label.text = "Инстаграм"
+         label.textColor = #colorLiteral(red: 0.4902775884, green: 0.6610349417, blue: 0.9130681157, alpha: 1)
+         label.font = UIFont(name: "Arial Bold", size: 15)
+         label.frame = CGRect(x: 20, y: 520, width: 100, height: 15)
+         return label
+     }()
+
+     private lazy var instagramTextField: UITextField = {
+         var textField = UITextField()
+         textField.frame = CGRect(x: 20, y: 540, width: 320, height: 30)
+         textField.placeholder = "Добавить"
+
+         textField.addTarget(self, action: #selector(instagramAlertAction), for: .allEditingEvents)
+         return textField
+     }()
+
+     override func viewDidLoad() {
+         super.viewDidLoad()
+         setupViews()
```

Comment:
```suggestion
          createDatePicker()
         configureAgePickerView()
         configureSexPickerView()
```
убрать в setupViews()